_reports/chay2199.md

```diff
@@ -0,0 +1,331 @@
+   - details: >
+      Sync Templates
+
+ - 27:
+   - repo: coala-json
+   - link: https://gitlab.com/coala/coala-json/commit/fd55822de4f1d9d0256cb3d4b7a85c338fbb5bfd
+   - details: >
+      Documentation of test result reports
+
+ - 28:
+   - repo: coala-json
+   - link: https://gitlab.com/coala/coala-json/commit/1b29998f1f7b922485d0fd03ed1cfe08cbab8d82
+   - details: >
+      Sync Templates
+
+ - 29:
+   - repo: coala-json
+   - link: https://gitlab.com/coala/coala-json/commit/e38dd3c35f4a6d17f6149da8e46ff6b35fcc555c
+   - details: >
+      Set documentation version
+
+ - 30:
+   - repo: coala-json
+   - link: https://gitlab.com/coala/coala-json/commit/4d2ccc569692b11b1fcfd2ea9077ae1f2d032882
+   - details: >
+      Publish coala-json version 0.1.0
+
+ - 31:
+   - repo: coala-json
+   - link: https://gitlab.com/coala/coala-json/commit/2a61b8a7c087ffa764326ba8a53f5f5c9a8a957f
+   - details: >
+      Remove redundant main_call function
+
+ - 32:
+   - repo: coala-json
+   - link: https://gitlab.com/coala/coala-json/commit/a4e8b25b62ab27c20b90a1a69533ba755f0fe977
+   - details: >
+      export PYTHONPATH to coala-json
+
+ - 33:
+   - repo: coala-json
+   - link: https://gitlab.com/coala/coala-json/commit/f1574f59e4d5c4a94123dd676c29ba95f7b873fd
+   - details: >
+      Add html-table output format
+
+ - 34:
+   - repo: coala-json
+   - link: https://gitlab.com/coala/coala-json/commit/1f4fbf07f668c557a1d032c3be2acd44293bfe28
+   - details: >
+      Add stdout output
+
+ - 35:
+   - repo: coala-json
+   - link: https://gitlab.com/coala/coala-json/commit/c4cb4dd598a7dfc867bc5a5e8906acc3bf242d3b
+   - details: >
+      JunitReporter.py: Change error tag to failure tag
+
+ - 36:
+   - repo: coala-json
+   - link: https://gitlab.com/coala/coala-json/commit/df856e5a7aa91ff126eff1f38c6c2b30338433d5
+   - details: >
+      coala-json version update 0.1.1
+
+ - 37:
+   - repo: coala-json
+   - link: https://gitlab.com/coala/coala-json/commit/6cc6ec867d7d77171cbf78884e173e6cc34abeb8
+   - details: >
+      Add coala-json to docs/requirements.txt
+
+ - 38:
+   - repo: coala-json
+   - link: https://gitlab.com/coala/coala-json/commit/2036499136abf87a72e34f108a302b2b9b5d2352
+   - details: >
+      Temporary fix for ruamel error
+
+ - 39:
+   - repo: coala-json
+   - link: https://gitlab.com/coala/coala-json/commit/e9dd53ccf602e57379ed6e270c52df6f74d62f0c
+   - details: >
+      Add badges
+
+ - 40:
+   - repo: coala-json
+   - link: https://gitlab.com/coala/coala-json/commit/249dfc598a29cc6aab197a1ebafe2f82098e3498
+   - details: >
+      README.rst: Update description
+
+ - 41:
+   - repo: mobans
+   - link: https://gitlab.com/coala/mobans/commit/a0205fd1da8ad3c147036fd7ee9a5218b9af6203
+   - details: >
+      coala-setup.py.jj2: Add missing commas
+
+ - 42:
+   - repo: mobans
+   - link: https://gitlab.com/coala/mobans/commit/aabd1098888e4983c92928c070b3bb8663adee58
+   - details: >
+      Sync Templates
+
```

Comment:
You should also mention the PR you opened up at pyflakes repo, and label it as unmerged